package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@transcend-io/penumbra",
-  "version": "4.10.0",
+  "version": "5.0.0",
```

Comment:
cool with major cuz its a big update, but conceptually wouldnt it be a minor since its a new feature with no breaking api changes?

Reply:
This changes the behavior for `penumbra.save`, so it should be backwards compatible with existing functionality. I guess 4.11 might be a better version